### ISSUE TITLE
[Feature Request] Switch tv input to Fire Tv

### ISSUE THREAD

**Issue by hamzaarbeed** [yuliskov/SmartTube#669]
Hello,
I think it is a good idea to let the app send a request to tv through CEC to switch tv input to fire tv, when sending a video from the phone to SmartTubeNext.

Thank you so much the app is awesome.

**Comment by Spamm00r** [comment]
Yeah i'd like this option too. Without it this is a dealbraker for many, who can't be bothered to switch inputs manually. 
The Stock Youtube APP on my  SmartTV switches to Youtube automatically, once you start casting to it. My parents prefer to use that and accept watching ads, instead of having to switch inputs every time to use SmartTubeNext.

Please add this feature urgently.

**Comment by UEF2** [comment]
The input switching also has the effect of turning on a TV that's otherwise off (via HDMI-CEC). Would be great to implement input switching as it'll probably do that too.

**Comment by nmt3325** [comment]
we need this feature.
